app/controllers/home_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
     @rentals = if @current_user.groups.where(name: 'admin').present?
                  Rental.all
                else
-                 @current_user.rentals
+                 Rental.where(renter: @current_user)
```

Comment:
Also, you removed `User.has_many :rentals`, so I think `user.rentals` doesn't work anymore.